src/VerifyU2FController.js

```diff
@@ -69,25 +79,62 @@ function (Okta, FormController, FormType, FooterSignout, Q) {
       className: 'verify-u2f-form',
       noCancelButton: true,
       save: _.partial(Okta.loc, 'verify.u2f.retry', 'login'),
+      noButtonBar: function () {
+        return !window.hasOwnProperty('u2f');
+      },
       modelEvents: {
         'error': '_showRetry',
         'request': '_hideRetry'
       },
 
-      formChildren: [
-        FormType.View({
-          View: '\
+      formChildren: function () {
```

Comment:
They look similar, but not similar and I am not sure if a lot of code can be moved to the shared view.